tidb-in-kubernetes/recovery-after-accidental-deletion.md

```diff
@@ -0,0 +1,14 @@
+---
+title: 集群意外删除后恢复
+category: troubleshooting
+---
+
+# 集群意外删除后恢复
```

Comment:
We can directly add this section to https://github.com/pingcap/docs-cn/blob/master/dev/how-to/maintain/backup-and-restore.md and add a subheading at the bottom. Could you please move it?